ngrok-java-native/src/main/java/com/ngrok/Runtime.java

```diff
@@ -70,17 +67,38 @@ private static boolean isPosixCompliant() {
     static native void init(Logger logger);
 
     static class Logger {
+        private static final org.slf4j.Logger logger = LoggerFactory.getLogger(Runtime.class);
         private static final String format = "[{}] {}";
 
-        public void log(String level, String target, String message) {
-            switch (level) {
-                case "TRACE" -> logger.trace(format, target, message);
-                case "DEBUG" -> logger.debug(format, target, message);
-                case "INFO" -> logger.info(format, target, message);
-                case "WARN" -> logger.warn(format, target, message);
-                case "ERROR" -> logger.error(format, target, message);
-                default -> logger.debug("{}: [{}] {}", level, target, message);
+        private static Logger instance;
+        private final Level level;
+
+        private Logger(Level level) {
+            this.level = level;
+        }
+
+        public static Logger Get() {
+            if (instance == null) {
+                String logLevel = System.getenv("NGROK_LOG_LEVEL");
```

Comment:
preferably, we shouldn't be depending on another env variable, just for ngrok logging 🤔